app/models/generic_event/per_court_cell_share_risk_assessment.rb

```diff
@@ -0,0 +1,15 @@
+class GenericEvent
+  class PerCourtCellShareRiskAssessment < GenericEvent
+    include PersonEscortRecordEventValidations
+
+    validates :location_id, presence: true
+
+    def location_id=(location_id)
+      details['location_id'] = location_id
+    end
+
+    def location_id
+      details['location_id']
+    end
+  end
```

Comment:
This looks like a good candidate for extracting to a shared concern perhaps - I see it used in a couple of events?

Reply:
Very good idea 👍 .